README.md

```diff
@@ -1,82 +1,88 @@
 # vuejs.org
+ 
+Bu site [hexo](http://hexo.io/) ile oluşturulmuştur. Site içeriği Markdown formatında yazılmıştır ve `src` içinde yer almaktadır. "Pull requests" istekleri çekin, hoşgeldiniz!
```

Comment:
> "Pull requests" istekleri çekin, hoşgeldiniz!

This should be "Her zaman katkıda bulunabilirsiniz."

Reply:
katıldığınız için teşekkür ederim Fatih bey, sayenizde merak saldım bu vujs'e baktım ki bizim dilimizde çeviriler yarım kalmış tamamlayalım diye düşündüm.